ProjectName/src/constants/Users.constant.ts

```diff
@@ -1,5 +1,5 @@
-import {Avatar} from '../types/Avatar';
-export const Avatars: Avatar[] = [
+import {User} from '../types/User';
```

Comment:
Please, leave blank line between last `import` statement and other pieces of code.